packages/cli/cloud/src/index.ts

```diff
@@ -0,0 +1,52 @@
+import { Command } from 'commander';
+import crypto from 'crypto';
+import deployProject from './deploy-project';
+import login from './login';
+import logout from './logout';
+import createProject from './create-project';
+import { CLIContext } from './types';
+import { getLocalConfig, saveLocalConfig } from './config/local';
+
+export const cli = {
+  deployProject,
+  login,
+  logout,
+  createProject,
+};
+
+const cloudCommands = [deployProject, login, logout];
+
+async function initCloudCLIConfig() {
+  const localConfig = getLocalConfig();
+
+  if (!localConfig.deviceId) {
+    localConfig.deviceId = crypto.randomUUID();
+  }
+
+  saveLocalConfig(localConfig);
```

Comment:
You are actually calling getLocalConfig & save in an async fn so they can also be async & use fs-extra async methods